descriptors/ModuleDescriptor-template.json

```diff
@@ -48,6 +48,21 @@
             "inventory-storage.service-points.item.get"
           ]
         },
+        {
+          "methods" : [ "POST" ],
+          "pathPattern" : "/bl-users/login-with-expiry",
+          "permissionsRequired" : [],
+          "modulePermissions" : [
+            "users.item.get",
+            "users.collection.get",
+            "perms.users.get",
+            "usergroups.item.get",
+            "inventory-storage.service-points-users.collection.get",
+            "inventory-storage.service-points-users.item.get",
+            "inventory-storage.service-points.collection.get",
+            "inventory-storage.service-points.item.get"
```

Comment:
Should there be a permission in this list to login / generate a token for the user?